lib/sassc/rails/template.rb

```diff
@@ -34,13 +34,36 @@ def call(input)
         }
       }.merge!(config_options) { |key, left, right| safe_merge(key, left, right) }
 
+      if Rails.application.config.assets.debug && Sprockets::VERSION.start_with?("4")
```

Comment:
Perhaps to be forward compatible this should be written in a way that it will still be enabled for sprockets 5 and other >4? It seems likely this will continue to be needed -- if it causes problems, sassc-rails will be broken with sprockets 5 either way, so doing this even in Sprockets 5+ seems the most likely to lead to things keeping working when/if a sprockets 5 is released.

Reply:
I checked and there are some options : 

```
Sprockets::VERSION.to_i >= 4
Gem.loaded_specs['sprockets'].version.segments.first >= 4
!Sprockets::VERSION.start_with?("3")
```

Sassc-rails doesn't support sprockets < 3 so, because the last line is the simplest, I think this is the better option.